schema-extended/Common.json

```diff
@@ -0,0 +1,308 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Common",
+  "description": "Common definitions.",
+  "type": "object",
+  "additionalProperties": false,
+  "definitions": {
+    "abstracts": {
+      "title": "List of abstracts",
+      "description": "The English and non-English versions of the abstract. The English abstract must be provided first.",
+      "type": "array",
+      "items": [
+        {
+          "type": "object",
+          "properties": {
+            "abstract": {
+              "type": "string",
+              "minLength": 1
+            },
+            "language": {
+              "const": "en"
+            }
+          },
+          "additionalProperties": false
+        }
+      ],
+      "additionalItems": {
+        "type": "object",
+        "properties": {
+          "abstract": {
+            "type": "string",
+            "minLength": 1
+          },
+          "language": {
+            "allOf": [
+              {
+                "$ref": "LanguageCode.json"
+              },
+              {
+                "not": {
+                  "enum": [
+                    "en"
+                  ]
+                }
+              }
+            ]
+          }
+        },
+        "additionalProperties": false
+      }
+    },
+    "arxiv_status": {
+      "title": "arXiv posting status",
+      "description": "The status of this document version on arXiv.",
+      "type": "string",
+      "enum": [
+        "posted",
+        "withdrawn"
+      ],
+      "minLength": 1
+    },
+    "author_id": {
+      "description": "arXiv author identifier, if available.",
+      "type": "string",
+      "pattern": "^[a-z]+_[a-z]{1}_[0-9]+$"
+    },
+    "comment": {
+      "title": "Comment",
+      "type": "string",
+      "minLength": 1
+    },
+    "conference_info": {
+      "title": "Conference information",
+      "type": "object",
+      "properties": {
+        "conference_titles": {
+          "title": "List of conference titles",
+          "type": "array",
+          "items": {
+            "type": "string",
+            "minLength": 1
+          },
+          "minItems": 1,
+          "uniqueItems": true
+        },
+        "conference_pid": {
+          "title": "Conference Persistent Identifier",
+          "$comment": "placeholder to support: https://www.crossref.org/working-groups/conferences-projects/",
+          "description": "The persistent identifier for the conference.",
+          "type": "string",
+          "minLength": 1
+        }
+      },
+      "required": [
+        "conference_titles"
+      ]
+    },
+    "collaboration": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "The fullest representation of a collaborations's name",
+          "allOf": [
+            {
+              "type": "string",
+              "minLength": 1
+            },
+            {
+              "not": {
```

Comment:
The collaboration definition now prevents the word "collaboration" from appearing in the collaboration name.

Reply:
This may be too strict.